### ISSUE TITLE
Add support whitelist in json file

### ISSUE THREAD

**Issue by leepowellcouk** [IBM/audit-ci#42]
It would be great if we could keep the whitelist advisory id's and module names in a json file of some kind on the file system.

**Comment by quinnturner** [comment]
Hi @leepowellcouk, my initial suggestion would be to create a script in `package.json`.

```
"scripts": {
  "audit-ci-default": "npm run audit-ci --low -a 605 604 660 -w dummypackage1 dummypackage2"
}
```

Then `npm run audit-ci-default`.

Does this satisfy your needs?

**Comment by leepowellcouk** [comment]
I guess it would work as a short term workaround - but we might want to have a centralised whitelist file we copy into our build pipeline for each of our projects rather than manually manage each projects npm scripts. Would you be open to a PR if I can find time to work on it?

**Comment by quinnturner** [comment]
I am OK with adding a config file. I would require it to be in the docs as well.

Perhaps introduce reading from a config file using `Yargs` since we already use that package: https://github.com/yargs/yargs/blob/master/docs/api.md#configobject. Since this supports JSON files, I suggest `audit-ci.json` to be the default file name, but have it configurable via the CLI. My initial suggestion for the CLI would be `-f` with `--file` alias.

**Comment by quinnturner** [comment]
Hi @leepowellcouk,

I created the PR #43. It parses the JSON file using Yargs. You are welcome to review it to ensure that it fulfills your needs!

**Comment by leepowellcouk** [comment]
Thank you! I will. Was hoping to get around to looking at this today but have been swamped.

**Comment by quinnturner** [comment]
Closing this since #43 was merged. Thanks for the support!

**Comment by quinnturner** [comment]
Released in #44 and published to NPM.
